plugins/leemons-plugin-leebrary/frontend/src/components/CardWrapper.js

```diff
@@ -64,14 +64,16 @@ const CardWrapper = ({
     const items = [];
 
     if (asset?.id) {
-      items.push({
-        icon: <AssignIcon />,
-        children: t('cardToolbar.covertToTask'),
-        onClick: (e) => {
-          e.stopPropagation();
-          history.push(`/private/tasks/library/create?from=leebrary&asset=${asset.id}`);
-        },
-      });
+      if (!asset?.fileType === 'bookmark') {
```

Comment:
@fermarinsanchez This condition is wrong

```suggestion
      if (asset?.fileType !== 'bookmark') {
```